packages/twenty-server/test/integration/metadata/suites/field-metadata/__snapshots__/create-one-field-metadata-select.integration-spec.ts.snap

```diff
@@ -0,0 +1,321 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Field metadata select creation tests group Create should fail with an invalid default value 1`] = `
+[
+  {
+    "extensions": {
+      "code": "BAD_USER_INPUT",
+    },
+    "message": "Default value for existing options is invalid: invalid",
+  },
+]
+`;
+
+exports[`Field metadata select creation tests group Create should fail with an unknown default value 1`] = `
+[
+  {
+    "extensions": {
+      "code": "BAD_USER_INPUT",
+    },
+    "message": "Default value for existing options is invalid: 'OPTION_424242'",
+  },
+]
+`;
+
+exports[`Field metadata select creation tests group Create should fail with comma in option label 1`] = `
+[
+  {
+    "extensions": {
+      "code": "BAD_USER_INPUT",
+    },
+    "message": "Label must not contain a comma",
+  },
+]
+`;
+
+exports[`Field metadata select creation tests group Create should fail with duplicated option ids 1`] = `
+[
+  {
+    "extensions": {
+      "code": "BAD_USER_INPUT",
+    },
+    "message": "Duplicated option id "fd1f11fd-3f05-4a33-bddf-800c3412ce98"",
```

Comment:
just for my knowledge, can snapshot include variables? does this id need to be hardcoded in the test case input?

Reply:
You can define specific [property-matchers](https://jestjs.io/docs/snapshot-testing#property-matchers), in case you dynamic data property such as date and ids ( you can still also extract these field and just check their definition before snapshoting remaining fields )